### PR TITLE
refactor-owner-login-screen

### PR DESCRIPTION
**Pull request by mahmmedn19**
- refactor-login-desgin
- create usecase validation and refactor loginOwner
- handle-validation in viewModel
- add keyboard options in textFileds in email and password

### CODE REVIEW

File: owner/src/main/java/org/the_chance/honeymart/ui/features/login/LoginViewModel.kt
Comment (AmeerAmjed) [inline owner/src/main/java/org/the_chance/honeymart/ui/features/login/LoginViewModel.kt:1]: This file have string hardcode can't translate.

File: owner/src/main/java/org/the_chance/honeymart/ui/features/signup/SignupScreen.kt [hunk 1]
```diff
@@ -81,28 +83,28 @@ fun SignupContent(
...
+                keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
             )
         }
 
         Column(
             verticalArrangement = Arrangement.SpaceBetween
         ) {
             HoneyFilledButton(
-                label = stringResource(R.string.continue_word),
+                label = stringResource(org.the_chance.design_system.R.string.continue_word),
```
Comment (AmeerAmjed) [inline owner/src/main/java/org/the_chance/honeymart/ui/features/signup/SignupScreen.kt:94]: Why change import can fix when set with import place or set with named is use another R

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]
Good job 👏

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]
Issues not resolved yet.

File: owner/src/main/java/org/the_chance/honeymart/ui/features/login/LoginViewModel.kt
Comment (Abdelrahman-Sedky) [reply owner/src/main/java/org/the_chance/honeymart/ui/features/login/LoginViewModel.kt:1]: We need to talk about it first

**Review by AmeerAmjed — APPROVED** [review]
Good job👏

**Pull request merged**